common-react/@dbeaver/ui-kit/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@dbeaver/ui-kit",
+  "type": "module",
+  "version": "0.1.0",
+  "description": "",
+  "license": "Apache-2.0",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "scripts": {
+    "build": "tsc -b",
+    "clean": "rimraf --glob dist",
+    "lint": "eslint ./src/ --ext .ts,.tsx"
+  },
+  "packageManager": "yarn@4.6.0",
+  "devDependencies": {
+    "@dbeaver/tsconfig": "workspace:^",
+    "@types/react": "^19",
+    "@types/react-dom": "^19",
+    "rimraf": "^6",
+    "typescript": "^5"
+  },
+  "dependencies": {
+    "@ariakit/react": "^0.4.15",
+    "@ladle/react": "^5.0.1",
+    "@tailwindcss/vite": "^4.0.3",
+    "@wroud/vite-plugin-asset-resolver": "^0.1.5",
```

Comment:
please keep only major version like `^5`
also check for dependencies that it's added correctly to `devDependencies` and `dependencies`